### ISSUE TITLE
border classes not working in v1.0.0-beta.1

### ISSUE THREAD

**Issue by lkoller** [tailwindcss/tailwindcss#895]
:wave: 

I'm using tailwind v1.0.0-beta.1 in a current project and it appears that border classes aren't working, I looked for an existing issue but couldn't find one.

`border-2 border-black border-solid ` this combination of tailwind classes does create a border all the way around the element

 but I can't seem to add a {side} signifier to the class to get just a bottom or right border.

`border-b-4 border-black border-solid` this still displays a border on all 4 sides

 I've had to revert to: 
```
.main-container {                                                                    
  @apply h-full flex;                                                                
  border-top: 1px solid #7a7586;                                                     
}
```

**Comment by adamwathan** [comment]
Hey! Can you create a repository that reproduces so I can help troubleshoot?

**Comment by mikemartin** [comment]
@lkoller I can confirm that the border utilities are working in my latest project running `1.0.0-beta.4`. If they still don't work after upgrading, I'd double check that you're not removing any border styles with PurgeCSS.

```
<div class="mt-4 border-2 border-black border-solid text-blue p-4">
   border-2 border-black border-solid
</div>

<div class="mt-4 border-b-4 border-black border-solid text-blue py-4">
   border-b-4 border-black border-solid
</div>
```
result:

<img width="508" alt="Screenshot 2019-05-13 15 10 25" src="https://user-images.githubusercontent.com/414211/57597151-8a268a00-7591-11e9-93e9-3aa2bc948ee8.png">

**Comment by adamwathan** [comment]
Closing for now until a reproduction is provided.

**Comment by wturrell** [comment]
I had a similar problem after [upgrading](https://tailwindcss.com/docs/upgrading-to-v1) from 0.7.4 to 1.0.1.

**TL;DR** was able to fix by rerunning`tailwind init`(after which, the issued immediately disappeared when re-running `tailwind build` using _either_ the empty, default tailwind.config.js OR my upgraded copy.

**Symptoms:**

This gave me a dotted border of even width on all sides, as intended:

`<p class="my-4"><a href="#" class="border border-dotted">Hello, world!</a></p>`

However, _this_ gave me the correct border on the bottom, but a bolder one the other three sides (rather than none, as I intended): 

`<p class="my-4"><a href="#" class="border-b border-dotted">Hello, world!</a></p>`

![tailwind-unwanted-border-example](https://user-images.githubusercontent.com/848021/57702810-bd7f1c80-7656-11e9-8d26-1cffc99f3c29.png)

I had run `npm install tailwindcss --save-dev` as per step 1, and there were no errors, so…?

npm v6.9.0, macOS 10.13.6
